packages/cypress/cypress/utils/oc_commands/scripts/mlflow_create_experiment.py

```
@@ -0,0 +1,25 @@
+experiment_name = sys.argv[3]
+
+data = json.dumps({"name": experiment_name}).encode()
+req = urllib.request.Request(
+    f"{mlflow_url}/api/2.0/mlflow/experiments/create",
+    data=data,
+    headers={
+        "Content-Type": "application/json",
+        "X-MLFLOW-WORKSPACE": workspace,
+    },
+)
+resp = urllib.request.urlopen(req)
```

Ruff (0.15.10)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)